### PR TITLE
Update Config files

### PR DESCRIPTION
**Pull request by Endorf**

### CODE REVIEW

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/VGSCheckoutAddCardConfig.kt [hunk 1]
```diff
@@ -4,16 +4,17 @@ import android.os.Parcel
...
 import com.verygoodsecurity.vgscheckout.model.VGSCheckoutEnvironment
 
 /**
- * Holds configuration with predefined setup for work with payment orchestration/multiplexing app.
+ * Holds configuration with predefined setup for work with payment orchestration app.
  *
- * @param accessToken multiplexing app access token.
- * @param tenantId payment orchestration tenant id.
+ * @param accessToken payment orchestration app access token.
+ * @param vaultID unique organization vault id.
```
Comment (DmytroDm) [inline vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/VGSCheckoutAddCardConfig.kt:17]: @Endorf Please check if name for id correct, should it be tenantId or vaultID?

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/VGSCheckoutAddCardConfig.kt [hunk 2]
```diff
@@ -52,30 +53,32 @@ class VGSCheckoutMultiplexingAddCardConfig private constructor(
...
      * @param isAnalyticsEnabled If true, checkout will send analytics events that helps to debug
      * issues if any occurs. Default value is true.
      *
-     * @throws com.verygoodsecurity.vgscheckout.exception.VGSCheckoutException if access token is not valid.
+     * @throws com.verygoodsecurity.vgscheckout.exception.VGSCheckoutJWTParseException if access token is not valid.
+     * @throws com.verygoodsecurity.vgscheckout.exception.VGSCheckoutJWTRestrictedRoleException if
+     * access token is contains restricted roles.
      */
     @JvmOverloads
-    @Throws(VGSCheckoutException::class)
```
Comment (DmytroDm) [inline vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/VGSCheckoutAddCardConfig.kt:66]: @Endorf Should be `VGSCheckoutException` like in `VGSCheckoutPaymentConfig`? 🤔

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/networking/VGSCheckoutPaymentRouteConfig.kt [hunk 3]
```diff
@@ -2,24 +2,24 @@ package com.verygoodsecurity.vgscheckout.config.networking
...
 
 /**
- * Multiplexing checkout route configuration. Only for internal use, as multiplexing request have
- * fixed structure.
+ * Payment orchestration checkout route configuration. Only for internal use,
+ * as Payment orchestration request have fixed structure.
  */
 @Parcelize
-class VGSCheckoutMultiplexingRouteConfig private constructor(
+class VGSCheckoutPaymentRouteConfig private constructor(
```
Comment (DmytroDm) [inline vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/networking/VGSCheckoutPaymentRouteConfig.kt:13]: @Endorf This config is used `VGSCheckoutAddCardConfig`, maybe we should add new type of route config? So every type of config will use its own route config?

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/ui/VGSCheckoutAddCardFormConfig.kt [hunk 4]
```diff
@@ -2,21 +2,21 @@ package com.verygoodsecurity.vgscheckout.config.ui
...
 import com.verygoodsecurity.vgscheckout.config.ui.core.VGSCheckoutFormValidationBehaviour
-import com.verygoodsecurity.vgscheckout.config.ui.view.address.VGSCheckoutMultiplexingBillingAddressOptions
-import com.verygoodsecurity.vgscheckout.config.ui.view.card.VGSCheckoutMultiplexingCardOptions
+import com.verygoodsecurity.vgscheckout.config.ui.view.address.VGSCheckoutPaymentBillingAddressOptions
+import com.verygoodsecurity.vgscheckout.config.ui.view.card.VGSCheckoutPaymentCardOptions
 import kotlinx.parcelize.Parcelize
 
 /**
- * Multiplexing flow checkout form configuration.
+ * Payment flow checkout form configuration.
```
Comment (DmytroDm) [inline vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/ui/VGSCheckoutAddCardFormConfig.kt:10]: @Endorf `Orchestration add card flow checkout form configuration`?

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/VGSCheckoutAddCardConfig.kt (hunk 2 above)
Comment (Endorf) [reply vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/VGSCheckoutAddCardConfig.kt:66]: Thanks! It might be an artifact after merging.

**Review by DmytroDm — APPROVED** [review]

**Pull request merged**